common/app/views/fragments/langAttributes.scala.html

```diff
@@ -0,0 +1,7 @@
+@(content: model.Content)(implicit request: RequestHeader)
+
+@defining(content.fields.lang.exists(!_.equals("en"))) { nonEnLang =>
```

Comment:
CAPI defaults to `en` if it does not recognise the language, therefore I thought it best not to explicitly declare the lang when `en` as CAPI may erroneously return `en` when a new language is used.